src/app/child-dev-project/attendance/attendance-details/attendance-details.component.html

```diff
@@ -47,6 +47,8 @@ <h1 mat-dialog-title>
         type="number"
         [value]="entity.countEventsPresent(forChild)"
         readonly
+        title="Number of days the child was present"
```

Comment:
any reason why you added these here while working on the location field? This looks familiar but probably we removed it for a reason (for example, these fields are read-only, so telling users to "enter number of .." doesn't make sense). I think we can skip adding this